services/data/data/dependencies/settings.py

```diff
@@ -163,9 +164,16 @@ def initialise_settings() -> None:
                   when=not_doc_unittest),
     )
     settings.validators.validate()
+    if not (settings.IS_CSW_SERVER or settings.IS_CSW_SERVER_DC or settings.IS_HDA_WEKEO):
+        raise Exception("No (meta)data connector is specified. At least one of the"
+                        "following env variables must be true: OEO_IS_CSW_SERVER,"
+                        " OEO_IS_CSW_SERVER_DC, OEO_IS_HDA_WEKEO.")
 
-    if settings.ENV_FOR_DYNACONF != "documentation":
+    if settings.IS_CSW_SERVER and settings.ENV_FOR_DYNACONF != "documentation":
```

Comment:
It's probably nicer to have one big if for "not documenation" and then the other ones inside (?)

Reply:
hmn yeah, I wanted to avoid nested if cycles, but I can update this